__test__/index.test.js

```diff
@@ -155,3 +181,26 @@ describe('hillchart@render', () => {
     expect(svg.getElementsByClassName('hill-chart-text')[1]).toBeUndefined();
   });
 });
+
+describe('hillchart@replaceData', () => {
```

Comment:
Have you replaced `describe('hillchart@render')`?

Reply:
No, that still exists at line 120